Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk
+WORKDIR /
+ADD target/conter-0.0.1-SNAPSHOT.jar conter-0.0.1-SNAPSHOT.jar
```

Comment:
To avoid having to change multiple lines in this `Dockerfile` as the version number of the artifact changes (as in fact it should on every CI build), give the copy of the JAR file in the image an unversioned name